fe/fe-core/src/main/java/org/apache/doris/service/FrontendServiceImpl.java

```diff
@@ -1664,7 +1664,7 @@ private boolean loadTxnCommitImpl(TLoadTxnCommitRequest request) throws UserExce
                     + request.isSetDbId() + " id: " + Long.toString(request.isSetDbId() ? request.getDbId() : 0)
                     + " fullDbName: " + fullDbName);
         }
-        long timeoutMs = request.isSetThriftRpcTimeoutMs() ? request.getThriftRpcTimeoutMs() / 2
+        long timeoutMs = request.isSetThriftRpcTimeoutMs() ? request.getThriftRpcTimeoutMs()
```

Comment:
Should reserve time for RPC transmission.

Reply:
Set value same as 9 / 10 of thrift RPC timeout?